src/_posts/languages/ruby/2000-01-01-getting-started-with-ruby.md

```diff
@@ -155,6 +155,23 @@ $ git push scalingo master
  <-- https://my-ruby-app.osc-fr1.scalingo.io -->
 ```
 
+## Database migrations
+
+To run a migration, you need to use a [post-deployment hook]({% post_url platform/app/2000-01-01-postdeploy-hook %}).
+
+The Ruby application starts up before the schema has changed, but does not receive a query.
+
+As a result, if operations are started before the migration has been executed, the application may load the schema `pre-migration`, and so a restart may be necessary after the migration, however short it may be.
+
+The best practice is to perform non-destructive migrations, where a column rename is performed in several steps, rather than an `ALTER TABLE` renaming a column.
+
+A good example in `strong_migrations` gem [documentation](https://github.com/ankane/strong_migrations#renaming-a-column).
+
+{% note %}
+  Running a migration on an application that then crashes on startup can be risky, so the application must start without the database migration.
+  As we make sure the application starts before running the hook, if it fails, `postdeploy` won't be run and database migration won't be executed either.
```

Comment:
suggestion: it's unclear to me what does "we" refers to
suggestion: shorter == easier to understand sentence

```suggestion
  Scalingo makes sure the application starts before running the hook. If the application does not start, the post-deployment hook won't run. Consequently the database migration won't be executed either.
```